recipe/meta.yaml

```diff
@@ -31,6 +31,7 @@ requirements:
     - pygeoif
     - python-dateutil
     - requests
+    - libxml2==2.9.3
```

Comment:
just add the `- icu 56.*` in the run dependencies, no `==` is needed.

BTW the backlog in Travis-CI is starting to clear and we probably won't need this once all the packages are updated to `icu 58.*`